src/main/java/org/apache/accumulo/access/Authorizations.java

```diff
@@ -33,13 +36,13 @@
  * @since 1.0.0
  */
 public final class Authorizations {
-  private final Set<String> authorizations;
+  private final SortedSet<String> authorizations;
 
-  private Authorizations(Set<String> authorizations) {
-    this.authorizations = Set.copyOf(authorizations);
+  private Authorizations(SortedSet<String> authorizations) {
+    this.authorizations = authorizations;
   }
 
-  public Set<String> asSet() {
+  public SortedSet<String> asSet() {
```

Comment:
This returns a mutable object that will change the internal state of the object when its mutated.  The javadoc states its immutable.